picard/coverart/__init__.py

```diff
@@ -130,7 +137,7 @@ def download_next_in_queue(self):
                     p = provider(self)
                     if p.enabled():
                         log.debug("Trying cover art provider %s ..." % name)
-                        ret = p.queue_downloads()
+                        ret = p.queue_images()
```

Comment:
I don't see a big problem either way. But I also don't see a real reason to break the API here and would provide a compatibility function anyway (and mark it deprecated). It is easy enough to get rid of that function a bit later.